code/modules/requisitions/fulton.dm

```diff
@@ -32,6 +37,9 @@
 		SSpoints.export_history += export_report
 	user.visible_message(span_notice("[user] finishes attaching [src] to [spirited_away] and activates it."),\
 	span_notice("You attach the pack to [spirited_away] and activate it. This looks like it will yield [export_report.points ? export_report.points : "no"] point[export_report.points == 1 ? "" : "s"]."), null, 5)
+	uses -= 1
+	if(zero_uses())
+		return TRUE
```

Comment:
do as splinter says. And why return TRUE anyway? You should just qdel when uses reach zero, but certainly not returning before qdel(spirited_away)